tests/Integration/PrestaShopBundle/Controller/Admin/Improve/International/CountryControllerTest.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Integration\PrestaShopBundle\Controller\Admin\Improve\International;
+
+use Symfony\Component\DomCrawler\Crawler;
+use Tests\Integration\Core\Form\IdentifiableObject\Handler\FormHandlerChecker;
+use Tests\Integration\PrestaShopBundle\Controller\FormGridControllerTestCase;
+use Tests\Integration\PrestaShopBundle\Controller\TestEntityDTO;
+
+class CountryControllerTest extends FormGridControllerTestCase
+{
+    public function testIndex(): int
+    {
+        $countries = $this->getEntitiesFromGrid();
+        $this->assertNotEmpty($countries);
+
+        return $countries->count();
+    }
+
+    /**
+     * @depends testIndex
+     *
+     * @param int $initialEntityCount
+     */
+    public function testCreate(int $initialEntityCount): void
+    {
+        $this->markTestSkipped('Not implemented');
+        // TODO: Implement when create action is created
+    }
+
+    /**
+     * @depends testCreate
+     *
+     * @param int $countryId
+     */
+    public function testEdit(int $countryId): void
+    {
+        $this->markTestSkipped('Not implemented');
+        // TODO: Implement when edit action is created
+    }
+
+    /**
+     * @depends testEdit
+     *
+     * @param int $countryId
+     */
+    public function testFilters(int $countryId): void
```

Comment:
Why not, it seems like filters are implemented so you should be able to test them based on the fixtures data